mslib/msui/qt5/ui_topview_mapappearance.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 
-# Form implementation generated from reading ui file '/home/zandar/Desktop/MSS/mslib/msui/ui/ui_topview_mapappearance.ui'
+# Form implementation generated from reading ui file 'ui_topview_mapappearance.ui'
```

Comment:
where is the ui file? please add it to.

Reply:
i think i have already pushed in this commit "[select feature and a ui](https://github.com/Open-MSS/MSS/pull/2428/commits/56a31d6d2252ef83e0084546d71cc788d8200bdb)"